mete0rama-orama/src/store/index.js

```diff
@@ -1,18 +1,47 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import axios from 'axios'
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
-    measurements: {}
+    measurements: [],
+    stationEndpoint: ''
   },
   mutations: {
-    updateMeasurements (state, measurements) {
+    SET_MEASUREMENTS (state, measurements) {
       state.measurements = measurements
+    },
+    SET_ENDPOINT (state, url) {
+      state.stationEndpoint = url
     }
   },
   actions: {
+    async fetchMeasurements ({ commit, state }) {
+      const endpoint = state.stationEndpoint
+      if (endpoint !== '') {
+        axios
+          .get(endpoint)
+          .then(response => {
+            commit('SET_MEASUREMENTS', response.data.data)
```

Comment:
Is the commit also an async call? If yes, we should add another `then` before resolving.

Reply:
no, committing the mutation is a synchronous call